search-commons/src/main/java/no/unit/nva/search2/resource/Constants.java

```diff
@@ -227,7 +229,15 @@ private static TermsAggregationBuilder instanceType() {
 
     private static TermsAggregationBuilder series() {
         return
-            branchBuilder(SERIES, ENTITY_DESCRIPTION, REFERENCE, PUBLICATION_CONTEXT, SERIES, TITLE, KEYWORD);
+            branchBuilder(SERIES, ENTITY_DESCRIPTION, REFERENCE, PUBLICATION_CONTEXT, SERIES, "issn", KEYWORD)
```

Comment:
"issn" as constant?

Reply:
Since this is the constant class, if it isn't repeated, it is only once declared and used.